packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx

```diff
@@ -299,7 +307,11 @@ const ProofOfAddressForm = observer(
                         <LeaveConfirm onDirty={is_mobile ? showForm : undefined} />
                         {form_state.should_show_form && (
                             <form noValidate className='account-form account-form_poa' onSubmit={handleSubmit}>
-                                <ThemedScrollbars height='572px' is_bypassed={!is_for_cfd_modal || is_mobile}>
+                                <ThemedScrollbars
+                                    height='572px'
```

Comment:
just a doubt, will size in rem work here?

Reply:
I think it might. Have to check to confirm

Reply:
It works. Will replace